src/scripts/main.js

```diff
@@ -1,8 +1,23 @@
 'use strict';
 
 const people = require('./lib/people');
+const table  = document.querySelector('.dashboard');
```

Comment:
```suggestion
const table = document.querySelector('.dashboard');
```
Пофикси ошибки
![image](https://user-images.githubusercontent.com/38065883/108068310-96931d00-706a-11eb-9867-33ed5da90307.png)